src/App.tsx

```diff
@@ -1,29 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { PaginationInfo } from './components/PaginationInfo';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const items: number[] = getNumbers(1, 42).map(n => n);
```

Comment:
The use of `.map(n => n)` is redundant here since it's just returning the same value. You can simply use `getNumbers(1, 42)`.